### PR TITLE
Added Home page

### PR DESCRIPTION
**Pull request by psychoactiv**
### Home Page

- Created a Home page where various product categories are listed.
- Made it responsive added a sidebar for smaller devices.
15 02 2022_12 56 32_REC

### CODE REVIEW

**Review by fahd23 — APPROVED** [review]
Looks good 🚀

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+
+.top-img {
+  width: 100%;
+  height: 100%;
+  grid-row-start: 1;
+  grid-row-end: 4;
+  grid-column-start: 1;
+  grid-column-end: 4;
+}
+
```
Comment (sushil-mod) [inline styles.css:166]: its correct, if possible use grid area - grid-area: row-start/col-start/row-end/col-end (if it works! )

**Review by sushil-mod — APPROVED** [review]
added one comment ,good to merge !

File: styles.css (hunk 1 above)
Comment (psychoactiv) [reply styles.css:166]: Ok, will try that.

**Pull request merged**